tests/xlsx/xlsx_import.test.ts

```diff
@@ -739,18 +739,75 @@ describe("Import xlsx data", () => {
     [
       "line",
       [
-        { dataRange: "Sheet1!B26:B35", backgroundColor: "#7030A0" },
+        {
+          dataRange: "Sheet1!B26:B35",
+          backgroundColor: "#7030A0",
+        },
         { dataRange: "Sheet1!C26:C35", backgroundColor: "#C65911" },
       ],
     ],
     [
       "bar",
       [
-        { dataRange: "Sheet1!B27:B35", backgroundColor: "#7030A0" },
+        {
+          dataRange: "Sheet1!B27:B35",
+          backgroundColor: "#7030A0",
+        },
         { dataRange: "Sheet1!C27:C35", backgroundColor: "#C65911" },
       ],
     ],
   ])("Can import charts %s with dataset colors", (chartType, chartDatasets) => {
+    const testSheet = getWorkbookSheet("jestCharts", convertedData)!;
+    const figure = testSheet.figures.find((figure) => figure.data.type === chartType);
+    const chartData = figure!.data as LineChartDefinition | BarChartDefinition;
+    expect(chartData.dataSets).toMatchObject(chartDatasets);
+  });
+
+  test.each([
+    [
+      "line",
+      [
+        {
+          dataRange: "Sheet1!B26:B35",
+          backgroundColor: "#7030A0",
+          trend: { type: "polynomial", order: 2, display: true, color: "#ED7D31" },
+        },
+        { dataRange: "Sheet1!C26:C35", backgroundColor: "#C65911" },
+      ],
+    ],
```

Comment:
```suggestion
    [
      "line",
      [{ trend: { type: "polynomial", order: 2, display: true, color: "#ED7D31" } },],
    ],
```
you could simplify the test a bit by only testing the trendlines